File: src/actions/python/encryptDataLocally.py
def encryptDataLocally(
    public_key: string, plaintext: string, policy: string, encryption_policy: string
):
    """Encrypt data locally"""
    cover_crypt = CoverCrypt()
    ciphertext = cover_crypt.encrypt(
        policy,
        encryption_policy,
        public_key,
        plaintext,
    )
    return ciphertext

File: src/actions/java/encryptDataLocally.java
//
// Encrypt data locally
//
public static void encryptDataLocally(String userDecryptionKey, String plaintext, Policy policy, MasterKeys masterKeys) {
    String encryptionPolicy = "department::HR && country::Germany";
    byte[] ciphertext = CoverCrypt.encrypt(policy, masterKeys.getPublicKey(), encryptionPolicy, plaintext,
    Optional.of(uid), Optional.empty());
}
